README.md

```diff
@@ -1 +1,14 @@
-# language-server
\ No newline at end of file
+#Ballerina Language Server
+A Language Server Implementation for [Ballerina](https://ballerinalang.org/)a general purpose, concurrent and strongly typed programming language with both textual and graphical syntax, optimized for integration. With this implementation we expose the language support for Ballerina in various IDEs by adhering to the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol).
+
+## Features
+In the current implementation we support _Auto completion_ and _program level symbol table support_ for Ballerina programs.
+
+## Installation
+You can find the VSCode plugin for Ballerina at VSCode [marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina)
```

Comment:
Since this is more of a library lets say LS integrated plugins can be found at VSCode & Composer

Reply:
Fixed